packages/webdriverio/src/commands/element/waitForExist.js

```diff
@@ -29,7 +29,7 @@
  * @param {Number=}  ms       time in ms (default: 500)
  * @param {Boolean=} reverse  if true it instead waits for the selector to not match any elements (default: false)
  * @param {String=}  error    if exists it overrides the default error message
- * @return {Boolean} true     if element exists (or doesn't if flag is set)
+ * @return {Promise<boolean>} true if element exists (or doesn't if flag is set)
```

Comment:
This is not true .. it always depends if you run WebdriverIO sync or asynchronously, We should not have this here.

Reply:
same